react/demos/theme/yarn.lock

```diff
@@ -10099,7 +10104,7 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
-typeface-open-sans@0.0.75, typeface-open-sans@^0.0.75:
+typeface-open-sans@^0.0.75:
```

Comment:
typeface-open-sans is still installed here because it's a dependency on the currently published version of the theme...it will get cleaned up once a new version of the theme is actually published and used here.